### PR TITLE
Suggest avoiding gclient

### PR DESCRIPTION
**Pull request by tanin47**
(I'm waiting for llvm to build... so I have time to cleanup our messages.)

### CODE REVIEW

File: tool/clusterfuzz/reproducers.py [hunk 1]
```diff
@@ -317,7 +317,12 @@ def reproduce_normal(self, iteration_max):
       # The crash signature validation is intentionally forgiving.
       if is_similar(new_signature, self.crash_signature):
         logger.info(common.colorize(
-            'The stacktrace seems similar to the original stacktrace.',
+            'The stacktrace seems similar to the original stacktrace.\n'
+            "Since you've reproduced the crash correctly, there are 2 tricks that might help "
+            'you move faster:\n'
+            '- You can save time by using `--disable-gclient` to avoid `gclient sync` and '
+            ' `gclient runhooks` in subsequential runs.\n'
+            '- In case of fixing the crash, you can use `--current` to avoid git-checkout.',
```
Comment (inferno-chromium) [inline tool/clusterfuzz/reproducers.py]: And mention that it uses tip-of-tree trunk.

Comment (inferno-chromium) [inline tool/clusterfuzz/reproducers.py]: Move this point up.

Comment (tanin47) [inline tool/clusterfuzz/reproducers.py]: That's a better word. Thanks for suggesting it.

Comment (tanin47) [inline tool/clusterfuzz/reproducers.py]: Done.

**Pull request merged**